common/database/connector.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * This singleton class provides a pooled Doctrine connection to consumers. All connection data
+ * is configurable via sqlconf.php.
+ *
+ * If needed, the instance can be used in a transactional context:
+ * <code>
+ *     $database = \common\database\Connector::Instance();
+ *     $entityManager = $database->entityManager;
+ *     $entityManager->getConnection()->beginTransaction();
+ *     try {
+ *         // Entity work here...
+ *         $entityManager->persist($someEntityToBePersisted);
+ *         $entityManager->flush();
+ *         $entityManager->getConnection()->commit();
+ *     } catch (Exception $e) {
+ *         $entityManager->getConnection()->rollBack();
+ *         throw $e;
+ *     }
+ * </code>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>.
+ *
+ * @package OpenEMR
+ * @author  Matthew Vita <matthewvita48@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+
+namespace common\database;
+
+use \Doctrine\ORM\Tools\Setup;
+use \Doctrine\ORM\EntityManager;
+
+include_once("../../interface/globals.php");
+require_once("../../library/sqlconf.php");
```

Comment:
if do always want to make sure sqlconf.php is included, then would be fine. But would use dirname(__FILE__) in the path to ensure nothing funny every happens.